CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.0.0 (Unreleased)
```

Comment:
If we're bumping this, we should probably bump the version in build.gradle to `2.0.0` as well.

Reply:
yep, it [already it](https://github.com/corretto/amazon-corretto-crypto-provider/blob/aws-lc/build.gradle#L12).